README.md

```diff
@@ -10,29 +10,94 @@ For general information about developing packages, see the Dart guide for
 and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages). 
 -->
+# miquido_flutter_lints
+##### This is collection of Flutter lints that we use and follow in [Miquido Software development company](https://www.miquido.com).
 
-This is collection of Flutter lints that we share and use in Miquido Sofware House. This is strict, large and rouboust collection beacuse we strive for high quality code and we want follow the same rules in all of our mobile apps.
+This is strict, large and robust collection beacuse we strive for high quality code, good practices and we want follow the same coding style in all our mobile apps.
+
+And we just looooove lots of lints.
 
 ## Features
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+- Using and following this linter makes your code better. FACTS.
 
 ## Getting started
 
-TODO: List prerequisites and provide or point to information on how to
-start using the package.
+We tested current configuration with this enviroment:
+```yaml
+environment:
+  sdk: ">=2.16.1 <3.0.0"
+  flutter: ^2.10.3
+```
+
+so we recommend to update your flutter project to met this conditions.
```

Comment:
"meet" instead of "met" 😛 
but I tried installing this package right now and there is a strict condition, that minimum Flutter version in project must be `2.10.3`. 
Therefore we should either change wording here (that it is actually required -not recommended)  or lower down the version. I guess we don't want to lower version though, because we would lose some lints because of that.

Reply:
yup we want to upgrade, not downgrade company projects. ;) changed to _required_